### DISCUSSION TITLE
Breeze theme not properly working with IOS 14.5 version

### DISCUSSION THREAD

**Discussion by niravGV** [breezefront/community#68] [1. Help]
We are facing problem with Breeze theme integration not working properly below IOS 14.5 version.
->  fields and the click of the elements not working
-> the model popups is not working.
-> some of working on ajax is not working properly.

Can you please help me with this problem.

**Note:- Also above the IOS 14.6 version its working smoothly everything.**

Thank you
Nirav.

**Comment by vovayatsyuk** [comment]
Unfortunately, we do not have a device with this OS to see what's wrong.

**Reply by niravGV** [reply]
Hello @vovayatsyuk 

We have checked our website in below software as well not working along with Actual device

You can just check with a [Lambda](https://www.lambdatest.com/) or [Browser Stack](https://www.browserstack.com/)
Its happen with these as well when selecting any device below 14.5 and any browser safari or chrome its happen.

For Ex:- **IPad Air 2019 13**

You can check it with a default breeze theme (https://breeze.swissupdemo.com/breeze_evolution/) as well not working properly.

Looking forward to hear your quick response.

Thanks
Nirav.

**Reply by vovayatsyuk** [reply]
Do you have account in Lambda or BrowserStack? Can you share a console output of what’s happening there when you trying to do something that’s not working?

**Reply by niravGV** [reply]
Yes have account with a BrowserStack and we have checked breeze theme broken, i have checked console but not have any error founded in that.

![Home-page](https://user-images.githubusercontent.com/30285024/222648860-5e89e05e-1dd8-4983-adb2-557e19cdd229.png)
![Homepage-footer](https://user-images.githubusercontent.com/30285024/222648869-8a72e658-1f4f-4f1e-a6fd-7ca4dc3e28aa.png)
![Customer-Account](https://user-images.githubusercontent.com/30285024/222648876-41aadf9b-8f74-4cf2-aec8-a7b16583fd51.png)
![Contacts](https://user-images.githubusercontent.com/30285024/222648880-c64de502-14d6-4b67-b6cb-2df5d007d998.png)

Hope this will be helpful for you.

**Reply by niravGV** [reply]
There are two different problems are there with lower version of the IOS browser.

-> Design broken problem
-> Some functionality problem like Model pop ups still checking for more problem.

**Reply by vovayatsyuk** [reply]
From what I see on the screenshots I think this Safari version doesn't support a lot of stuff we are using (CSS properties, modern selectors: `:where`, `:has`)

I don't want to investigate it further since it's an outdated browser version.

p.s. What's driving you to support it?
